samples/DocumentTaskExample/Program.cs

```diff
@@ -64,9 +64,11 @@ private static void AddDocumentTask(string filePath)
 
         private static void AddNewTask(MainDocumentPart mdp, string title, User assignee, User assigner)
         {
-            // TODO: userids, providers and names will have to be programmatically accessed via directory services.
-            // These are just examples to show what is expected.
+#pragma warning disable TODO // TODO
```

Comment:
What's this for?

Reply:
If the TODO warning isn't disabled, the build fails with stylecop errors. I wanted to leave the TODO in so users would know this was dummy data, so I disabled the error for this line. Thinking about it now, I should have just deleted "TODO" and left the rest of the comment...